util/Setup/EnvironmentFileBuilder.cs

```diff
@@ -85,7 +75,7 @@ private void Init()
             _globalOverrideValues = new Dictionary<string, string>
             {
                 ["globalSettings__baseServiceUri__vault"] = _context.Config.Url,
-                ["globalSettings__sqlServer__connectionString"] = $"\"{dbConnectionString}\"",
+                ["globalSettings__sqlServer__connectionString"] = $"\'{dbConnectionString}\'",
```

Comment:
With the `'` swap from `"` around the connection string, you no longer need the escape `\` characters.
```suggestion
                ["globalSettings__sqlServer__connectionString"] = $"'{dbConnectionString}'",
```